src/Uno.UI/UI/Xaml/UIElement.iOS.cs

```diff
@@ -459,9 +459,26 @@ public override void TouchesCancelled(NSSet touches, UIEvent evt)
 					CanBubbleNatively = true,
 					OriginalSource = this
 				};
-				var isHandledInManaged = RaiseEvent(PointerCanceledEvent, args);
 
-				if (!isHandledInManaged)
+				// As soon as a gesture is recognized by a 'GestureRecognizer' (cf. Tap and DoubleTap events)
+				// the touches are "cancelled" and this method is invoked. So until we remove the GestureRecognizers
+				// for those events, we handle the "Cancelled" similarly as the "Ended" and we raise the "Release".
+				var pointerEventIsHandledInManaged = false;
+				if (IsPointerCaptured || evt.IsTouchInView(this))
```

Comment:
The test for this one is to put a Button inside a ScrollViewer, press on the Button, and then start to scroll. The Button's Click shouldn't be raised.

Reply:
Did you make this test?

Reply:
As the `Click` is linked to the `Tapped`, which is raised by the `GestureRecognizer`, it won't be affected by this commit. However yes I've validated, it works properly.

BTW I noticed that we don't even receive the `PointerPressed` nor `PointerReleased` on a `Button` on Windows (FYI @carldebilly ). This should be included in final fix of #864